docs/quick-start/transmitters/firmware-version.md

```diff
@@ -0,0 +1,157 @@
+        - http://10.0.0.1/ -If you have connected to the `ExpressLRS TX` Access Point
+            
+            !!! Note
+                This IP address is also often being used by some routers. Computers with an ethernet connection to their routers will find that this page doesn't load the ExpressLRS Web UI and instead loads their router or ISP modem configuration dashboards.
+
+                We recommend disconnecting from the network first, or try a different device instead, e.g. smartphone or tablet.
+        
+        - http://elrs_tx.local - If you have previously set your Home WiFi SSID and Password into your TX Module when you flashed it.
+
+            !!! Note
+                Some computers and networks cannot resolve this address if MDNS is not working. 
+
+            ??? tip "Use the IP Address instead!"
+                === "The `arp` Command"
+
+                    1. Open up a Command Prompt window on your computer.
+                    2. Execute the command `arp -a` which will list all the devices in the Network.
+                    3. Use each of the IP Addresses marked as `Dynamic` as URL into your Browser until you get to the ExpressLRS Web UI.
+
+                === "Router DHCP List"
+                    1. Log in into your Router dashboard.
+                    2. Check the DHCP List and look for the "elrs" device.
+                    3. Take note of the IP Address given by your router.
+                    4. Use this IP address into your Browser as the URL.
+
+    4. The Banner of the Web UI should tell you the firmware version flashed into your Tx module.
+
+        -Insert Image of the new Web UI Banner for TX-
+
+<hr />
+
+Once you have determined the firmware version that's currently flashed into your Transmitter module, you can now decide to whether keep using it as is or update to the latest version.
+
+!!! Info "Versioning Scheme"
+    The ExpressLRS Firmware Versioning scheme is based on the [semantic versioning scheme](https://semver.org/).
+
+    A Version is defined or formatted as "Major.Minor.Patch"
+
+        - major = major new feature and/or incompatible changes
+        - minor = minor features or enhancements and/or new targets
+        - patch = bug-fixes
+    
+    That first number in the Version string should match between a TX Module and a Receiver.
+    Examples:
+
+    - a TX Module with version string of 3.1.2 will sync and work with a Receiver with firmware version 3.0.1
+    - a TX Module with version string of 3.2.0 will NOT sync or bind with a Receiver with firmware version 2.4.0
+
+
+For the latest ExpressLRS firmware version, check the [Releases page in Git Hub](https://github.com/ExpressLRS/ExpressLRS/releases)
+
+## I Know the Version. What Now?
+
+=== "I have an old version"
+
+    See the [next page](../transmitters/updating.md) for the typical updating procedure or use the links below to find the updating instructions specific to your TX Module.
+
+    -links-
```

Comment:
remove

Reply:
My intention for this section is to provide a "directory" for all individual hardware pages, with photo links.. 

I was also thinking of using the search bar here so users can type in their hardware, and the link to the updating guide will show up. Seems mkdocs doesn't allow the search bar somewhere else than the header though.